app/assets/stylesheets/application.scss

```diff
@@ -225,6 +225,74 @@ main {
   font-size: 15px;
 }
 
+.nav-tabs .nav-link {
+  border: none;
+}
+
+.nav-tabs .nav-link.active {
+  background-color: transparent;
+  font-weight: bold;
+  border-bottom: 3px solid #696df2;
+}
+
+.bg-purple {
+  background-color: rgb(226, 225, 253);
+}
+
+.system-wrapper {
+  display: grid;
+  grid-template-columns: 3fr 4fr 4fr 2fr 1fr;
+  grid-gap: 1em;
+  grid-auto-rows: minmax(100px, auto);
+}
+
+.system-wrapper img {
+  width: 100%;
+  height: 100%;
+}
+
+.system-box:nth-child(1) {
+  border-bottom: 1px solid #ccc;
+}
+
+.system-box {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  grid-auto-rows: minmax(100px, auto);
+  justify-content: center;
+}
+
+.system-box-title{
+  font-size: 15px;
+}
+
+.system-box-five {
+  position: absolute;
+  top: 10%;
+  right: 2%;
+}
+
+.status-on {
```

Comment:
remove .status-on and .status-off and instead use bootstrap classes "text-red" and "text-green""